components/dashboard/src/admin/WorkspacesSearch.tsx

```diff
@@ -105,6 +114,10 @@ export function WorkspaceSearch(props: Props) {
                 <button disabled={searching} onClick={search}>Search</button>
             </div>
         </div>
+        <div className={'flex rounded-xl bg-gray-200 dark:bg-gray-800 text-gray-600 dark:text-gray-400 p-2 w-2/3 mb-2'}>
+            <img className="w-4 h-4 m-1 ml-2 mr-4" alt="info" src={info} />
+            <span>Please enter complete IDs - this search does not perform partial-matching.</span>
```

Comment:
This one hurt! Opened https://github.com/gitpod-io/gitpod/issues/8701 to improve this.

<img width="686" alt="Screenshot 2022-03-09 at 10 52 00 PM" src="https://user-images.githubusercontent.com/120486/157533990-8bb3b010-c8f8-4219-a07c-0298a8da8594.png">